examples/integrations/langchain4j/coffee-shop-assistant-se/src/main/java/io/helidon/examples/integrations/langchain4j/se/coffee/shop/assistant/rest/ChatBotService.java

```diff
@@ -41,6 +50,11 @@ public class ChatBotService implements HttpService {
     @Service.Inject
     public ChatBotService(ChatAiService chatAiService) {
         this.chatAiService = chatAiService;
+        this.chatLanguageModel = OpenAiChatModel.builder()
+                .apiKey("demo")
+                .modelName(GPT_4_O_MINI)
+                .listeners(List.of(new MetricsChatModelListener()))
+                .build();
```

Comment:
This is bypassing our integration, we need to add it as a new feature

Reply:
Agreed. We decided to do this only in 4.3 but show case as example in 4.2.